gatling-core/src/main/resources/gatling-defaults.conf

```diff
@@ -105,6 +105,7 @@ gatling {
   }
   jms {
     replyTimeoutScanPeriod = 1000  # scan period for timedout reply messages
+    listenerThreadCount = 1        # listener thread count for receiveing and dispatching jms messages
```

Comment:
I think such parameter would be more visible for users if passed from JmsProtocol instead of config.